SketchwareMaterialRedesign

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is that file? Seems unwanted file. Can you remove it?

Reply:
idk, I didn't make such a file

Reply:
it was made automatically